kano_i18n/assets.py

```diff
@@ -0,0 +1,19 @@
+# assets.py
+#
+# Copyright (C) 2015 Kano Computing Ltd.
+# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
+#
+# Utility function for finding non-text translated assets
+#
+
+import locale
+import os
+
+
+def get_path(base, relative, default='en_US'):
+    (code, _) = locale.getdefaultlocale()
```

Comment:
Given that this function will almost certainly be called from an i18n app, `_` will be defined as the translate function so might be best not to overwrite it.

Reply:
Excellent point.